appinventor/blocklyeditor/src/generators/yail/procedures.js

```diff
@@ -85,3 +85,36 @@ Blockly.Yail['procedures_callreturn'] = function() {
       + Blockly.Yail.YAIL_CLOSE_COMBINATION;
   return [ code, Blockly.Yail.ORDER_ATOMIC ];
 };
+
+Blockly.Yail['map_list'] = function() {

```

Comment:
Comment out for now and move to lists.js.

Reply:
Fixed